tark/tark/settings/env.py

```diff
@@ -19,5 +19,5 @@
 
 ENV = os.getenv('DJANGO_ENVIRONMENT', 'development')
 DEV_ENV = ENV == 'development'
-TEST_ENV = ENV == 'staging'
-PROD_ENV = ENV == 'production'
+STAGING_ENV = ENV == 'staging'
+PROD_ENV = ENV == 'primary'
```

Comment:
Why change 'production' to 'primary'?  Could we update "ENV setup" on this page to match? https://www.ebi.ac.uk/seqdb/confluence/pages/viewpage.action?pageId=111295328

Reply:
That confluence page will go. We will have a new setup/deployment SOP(WIP)